_extras/checkout.md

```diff
@@ -49,40 +49,52 @@ and must be requested before the three months are up. To request an extension, p
 ## Part 1: Submit a Small Contribution to One of Our Lessons
 <a name="lesson-change"></a>
 
-Our lessons are maintained and improved by the people who teach them,
-so this part of the checkout procedure gives you a chance to familiarize yourself
+Our lessons and lesson infrastructure are maintained and improved by the people who use them.
+With this part of the checkout procedure, you will familiarize yourself
 with our collaborative curriculum development process.
 
-Trainees must make a contribution to a lesson's content, exercises, or instructor's guide by doing one of the following:
+### Examples of Contributions
 
-1. Providing substantive feedback on an existing issue or pull request (preferred).
-2. Submitting a change to fix an existing issue.
-3. Proof-reading a lesson and adding a new issue describing something to be improved.
+Trainees must make a contribution to any GitHub repository in The Carpentries organization that is actively maintained and supports a lesson (e.g. [LC Open Refine](https://github.com/LibraryCarpentry/lc-open-refine/), [SWC Unix Shell](http://swcarpentry.github.io/shell-novice/)) or lesson infrastructure (e.g. the [handbook](https://github.com/carpentries/handbook), [workshop template](https://github.com/carpentries/workshop-template)). 
 
-Contributions to Data Carpentry and Library Carpentry materials may be submitted through GitHub or by [email](mailto:{{ site.email }}). Contributions to Software Carpentry materials must be submitted through GitHub.
+We **encourage** the following types of contributions:
```

Comment:
Suggested by @libcce in Slack was that we could direct people to issues tagged as 'Good first issue'. I think this is a good idea and wonder if it might be included here?

I generally the way I use this tag is on an issue which looks ready for a simple PR but not sure if this is common across carpentry lesson repos